ext/standard/random.c

```diff
@@ -158,19 +159,19 @@ PHP_FUNCTION(random_int)
 	zend_ulong umax;
 	zend_ulong result;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ll", &min, &max) == FAILURE) {
+	if (zend_parse_parameters_throw(ZEND_NUM_ARGS(), "ll", &min, &max) == FAILURE) {
 		return;
 	}
 
 	if (min >= max) {
```

Comment:
@ircmaxell Hmm, I sometimes have cases like where I get an unsigned integer and then want a random number between 0 and that integer… surely that integer can be zero too... Yes, it's not a random value then, but actually the most trivial case. No (?) reason to exclude it, I think.

Would be a bit dumb to have to `function better_random_int($a, $b) { return $a === (int) $b ? random_int() : $a; }` just for that...

After all, the point is to pick a number on a range. `[0, 0]` is valid. Just like `[0, 1]` is and `[0, -1]` isn't.